src/components/WorkOrders/CloseWorkOrderFormComponents/FollowOnRequestTypeOfWorkForm.js

```diff
@@ -14,6 +14,8 @@ const FollowOnRequestTypeOfWorkForm = (props) => {
     clearErrors,
     watch,
     followOnData,
+    hasWhiteBackground,
+    isGrid,
```

Comment:
Adding these style related props is going to become unmaintainable quickly. Could we instead use a class prop, which combines them all.

For example:
```
import cx from 'classnames'
...
<
  class={cx("checkbox-classes", classProp)}
/>



```